frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -579,6 +579,7 @@
   "Create ObjectBucketClaim": "Create ObjectBucketClaim",
   "Edit YAML": "Edit YAML",
   "Attach to Deployment": "Attach to Deployment",
+  "Disabled because the Object Bucket Claim is being deleted.": "Disabled because the Object Bucket Claim is being deleted.",
```

Comment:
```suggestion
  "Disabled because the ObjectBucketClaim is being deleted.": "Disabled because the Object Bucket Claim is being deleted.",
```

?

Reply:
Is PascalCase user friendly?

Reply:
Nevermind, I can see that we've used `ObjectBucketClaim` throughout the i18n `json`s, making the change. 👍🏼